modules/src/main/assembly/dist.xml

```diff
@@ -140,6 +145,15 @@
             <unpack>false</unpack>
             <includes>
                 <include>org.hibernate:hibernate-search-elasticsearch</include>
+            </includes>
+        </dependencySet>
+
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <outputDirectory>org/hibernate/search/elasticsearch-client/${hibernate.search.version}</outputDirectory>
+            <useTransitiveFiltering>false</useTransitiveFiltering>
+            <unpack>false</unpack>
+            <includes>
                 <include>com.google.code.gson:gson</include>
```

Comment:
Good point, but I don't want to expose it to our users.

I'll move gson to its own module then..